DigitalLearningSolutions.Web/Views/ResetPassword/Index.cshtml

```diff
@@ -1 +1,45 @@
-<h1>Reset Password</h1>
+@using DigitalLearningSolutions.Web.ViewModels.Common
+@model PasswordViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Reset Your Password" : "Reset Your Password";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <form method="post" asp-action="Index">
+
+      @if (errorHasOccurred) {
+        <partial name="_ErrorSummary"/>

```

Comment:
I think it would be worth replacing this with the new error summary view component. Would probably look something like this:
<vc:error-summary order-of-property-names="@(new[] { nameof(Model.Password), nameof(Model.ConfirmPassword) })"/>